LocalSettings.php

```diff
@@ -178,17 +178,31 @@
 			'rangeblock' => false,
 			'tag' => true,
 		),
+		'wikicanadawiki' => array(
```

Comment:
Can't you just use rangeblock => true, and remove the rest as they're already default values?